README.md

```diff
@@ -1,3 +1,208 @@
-# Migrated to New Github Repo https://github.com/angel-one/smartapi-python
-For latest updates and bug fixes please refer to the new git repo.
+# SMARTAPI-PYTHON
+
+SMARTAPI-PYTHON is a Python library for dealing AMX,that is a set of REST-like HTTP APIs that expose many capabilities required to build stock market investment and trading platforms. It lets you execute orders in real time.
```

Comment:
we need not mention AMX to our end customers. We can simply write interacting with Angel trading platform